### PR TITLE
Update foreman_remote_execution to 1.2.1 (DEB)

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

We moved the smart_proxy_remote_execution_ssh_core and part of foreman_remote_execution
to foreman_tasks_core and foreman_remote_execution_core to be able to use rex
without a proxy and reuse the code for foreman-ansible. No installer change
should be necessary.

### CODE REVIEW

File: plugins/ruby-foreman-tasks/debian/control [hunk 1]
```diff
@@ -2,9 +2,9 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.13.0~rc1), foreman-sqlite3 (>= 1.13.0~rc1), foreman-assets
+Build-Depends: debhelper, foreman (>= 1.13.0~rc1), foreman-sqlite3 (>= 1.13.0~rc1), foreman-assets,
```
Comment (mmoll) [inline plugins/ruby-foreman-tasks/debian/control]: extra comma

**Comment by iNecas** [comment]
I've extracted the smart proxy changes into https://github.com/theforeman/foreman-packaging/pull/1330, leaving this PR just for foreman server part

File: plugins/ruby-foreman-tasks/debian/control (hunk 1 above)
Comment (iNecas) [inline plugins/ruby-foreman-tasks/debian/control]: updated

**Comment by mmoll** [comment]
[test] again

**Comment by mmoll** [comment]
merged with the rex version fixed up in plugins/ruby-foreman-remote-execution/foreman_remote_execution.rb, díky @iNecas!

**Pull request closed without merging**